src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/publicApi/PublicLicencePolicyService.kt

```diff
@@ -16,4 +16,9 @@ class PublicLicencePolicyService(
 
     return policy.transformToPublicLicencePolicy()
   }
+
```

Comment:
Not a biggie but can be shortened to `return licencePolicyService.currentPolicy().transformToPublicLicencePolicy()`

Reply:
fixed